x-pack/test/functional/services/ml/api.ts

```diff
@@ -1065,6 +1065,15 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
       });
     },
 
+    async getGroups(space?: string) {
+      const { body, status } = await kbnSupertest
+        .get(`${space ? `/s/${space}` : ''}/internal/ml/jobs/groups`)
```

Comment:
Out of scope of this PR but in the future, we should rename this endpoint to indicate it also returns calendars 🤔

Reply:
> to indicate it also returns calendars

I'm not sure if I completely agree with this statement. The endpoint returns a list of every group, and for each group it lists the IDs of the jobs and calendars with which the group is associated.
So it's true that it contains calendar IDs, but not all calendar IDs, only the ones which have been associated with a group.

e.g.

```
[
  {
    "id": "anomaly-alerting",
    "jobIds": [
      "rt-anomaly-mean-value"
    ],
    "calendarIds": [
      "cal_1"
    ]
  },
  {
    "id": "real-time",
    "jobIds": [
      "rt-anomaly-mean-value"
    ],
    "calendarIds": [
      "cal_2"
    ]
  }
]
```

In my opinion these are "groups" and the endpoint is named correctly.
I wouldn't know what else to name it if we wanted to reflect the exact structure of the response.